src/hotspot/share/gc/shared/preservedMarks.hpp

```diff
@@ -65,6 +66,8 @@ class PreservedMarks {
   // to their forwarding location stored in the mark.
   void adjust_during_full_gc();
```

Comment:
Does this method have any uses? Should those be updated to use `forwarding`? If not, it should be removed for safety?

Reply:
Actually not, except in a unit test. This test should be written to use a forwarding instead, but this probably requires a mockable/abstract forwarding that can be plugged in for the purpose of the test. I added TODOs and postpone this to later.